src/pages/Login/Login.js

```diff
@@ -1,5 +1,96 @@
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+
 const Login = () => {
-  return <div>Login</div>;
+  const navigate = useNavigate();
+
+  function kakaLogin() {
+    window.Kakao.Auth.login({
+      scope: 'profile_nickname, profile_image',
+      success: function (res) {
+        window.Kakao.API.request({
+          url: '/v2/user/me',
+          success: res => {
+            fetch('http://10.58.1.126:8000/users/signup', {
+              method: 'GET',
+              headers: {
+                Authorization: res.access_token,
+              },
+            })
+              .then(res => res.json())
+              .then(res => {
+                if (res.token) {
+                  localStorage.setItem('token', res.token);
+                  goToMain();
+                } else {
+                  alert('로그인 실패');
+                }
+              });
+          },
+        });
+      },
+    });
+  }
+
+  const goToMain = () => {
+    alert('로그인에 성공하였습니다.');
+    navigate('/');
+  };
+
+  return (
+    <LoginSection>
+      <LoginBox>
+        <MainImg alt="로그인배경이미지" src="images\login background img.jpg" />
```

Comment:
```suggestion
        <MainImg alt="로그인배경이미지" src="/images/login background img.jpg" />
```

절대경로로 적어주세요! 상대경로로 지정하게 되면 path의 depth가 깊어졌을때 원하는대로 public 폴더에 접근을 하지 못합니다!

예시) 상대경로로 작성했을 시
- http://localhost:3000/main => 이경우에는 상대경로로 작성해도 public 폴더로 정상 인식해서 정상동작
- http://localhost:3000/detail/1 => 이경우에는 path의 depth가 깊어져서 상대경로로 지정하게되면 pubilc/detail 폴더안에서 찾으려고 하기 때문에 제대로 동작하지 않음.
- 경로가 계속 "/"가 아니라 "\"로 처리된 것이 많이 보이는데 별다른 이유가 있나요?

Reply:
아니요 별다른 이유 없고 그냥 그렇게 습관이 된 것 같습니다 수정하겠습니다!